### PR TITLE
More graph db tests

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: integration-tests/features/steps/gremlin.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def valid_gremlin_response(context):
 
 @then('I should get zero vertexes')
 @then('I should get {num:d} vertexes')
-def check_vertexes_cound(context, num=0):
+def check_vertexes_count(context, num=0):
```
Comment (jpopelka) [inline integration-tests/features/steps/gremlin.py]: There's one more `check_vertexes_cound()` function below.

File: integration-tests/features/steps/gremlin.py [hunk 2]
```diff
@@ -136,6 +136,35 @@ def check_vertexes_cound(context):
     assert vertexes > 0, "Expected at least one vertex, but got zero instead"
 
 
+def get_node_value_from_properties_returned_by_gremlin(context, node_name):
+    """Try to retrieve node value from the 'properties' node returned by Gremlin."""
+    data, meta = get_results_from_gremlin(context)
+    assert len(data) == 1, "Expected preciselly one vertex with package data"
```
Comment (jpopelka) [inline integration-tests/features/steps/gremlin.py]: precisely

File: integration-tests/features/steps/gremlin.py [hunk 3]
```diff
@@ -136,6 +136,35 @@ def check_vertexes_cound(context):
...
+    """Check the package name in Gremlin response."""
+    name = get_node_value_from_properties_returned_by_gremlin(context, "name")
+    assert name == package, \
+        "Returned package name '{name}' is different from expected name '{package}'" \
+        .format(name=name, package=package)
+
+
+@then('I should find the ecosystem {ecosystem} name in the Gremlin response')
+def check_ecosystem_name(context, ecosystem):
+    """Check the package name in Gremlin response."""
```
Comment (jpopelka) [inline integration-tests/features/steps/gremlin.py]: package -> ecosystem ?

File: integration-tests/features/steps/gremlin.py [hunk 4]
```diff
@@ -136,6 +136,35 @@ def check_vertexes_cound(context):
...
+    name = get_node_value_from_properties_returned_by_gremlin(context, "ecosystem")
```
Comment (jpopelka) [inline integration-tests/features/steps/gremlin.py]: In `check_package_name()` you call
`name = get_node_value_from_properties_returned_by_gremlin(context, "name")`
so something tells me that you want to do the same here.

(or
`name = get_node_value_from_properties_returned_by_gremlin(context, "package")`
there ?)

Comment (tisnik) [reply integration-tests/features/steps/gremlin.py]: the 'ecosystem' node is assigned to the vertex and we need to check its' value:

```
{   'requestId': '735de202-75f6-4d21-85e4-02854715c59a',
    'result': {   'data': [   {   'id': 6144057480,
                                    'label': 'vertex',
                                    'properties': {   'ecosystem': [   {   'id': '7c6raj5-2tm0h3c-sl',
                                                                             'value': 'npm'}],
```

**Pull request merged**